game/sound/CMakeLists.txt

```diff
@@ -26,6 +26,8 @@ target_link_libraries(sound PRIVATE fmt cubeb common)
 add_executable(sndplay 989snd/sndplay.cpp)
 if(WIN32)
     target_link_libraries(sndplay PRIVATE sound cubeb)
+elseif(APPLE)
+    target_link_libraries(sndplay PRIVATE sound cubeb)
```

Comment:
`-lstdc++fs` was not found by my linker, and `#include <filesystem>` seems to work fine on macOS anyway.